src/scripts/main.js

```diff
@@ -1,7 +1,44 @@
 'use strict';
+// document.createElement('div');
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification');
+
+  switch (type) {
+    case 'success':
+      notification.classList.add('success');
+      break;
+    case 'error':
+      notification.classList.add('error');
+      break;
+    case 'warning':
+      notification.classList.add('warning');
+      break;
+    default:
+      break;
+  }
+
+  notification.getBoundingClientRect();
+  notification.style.top = posTop + 'px';
+  notification.style.right = posRight + 'px';
+
+  const notTitle = document.createElement('h2');
+
+  notTitle.textContent = title;
+  notTitle.classList.add('title');
+
+  const notDescription = document.createElement('p');
+
+  notDescription.textContent = description;
+
+  notification.append(notTitle);
+  notification.append(notDescription);
+
+  document.body.append(notification);
+
+  setTimeout(() => (notification.style.display = 'none'), 2000);
```

Comment:
Instead of setting `notification.style.display = 'none'`, consider removing the notification from the DOM entirely using `notification.remove()`. This will prevent unnecessary elements from remaining in the DOM.